pkgs/by-name/ar/arrpc/package.nix

```
@@ -1,31 +1,39 @@
-{ lib
-, buildNpmPackage
-, fetchFromGitHub
-}: buildNpmPackage rec {
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+buildNpmPackage rec {
   pname = "arrpc";
-  version = "3.4.0";
+  version = "3.5.0";
 
   src = fetchFromGitHub {
     owner = "OpenAsar";
     repo = "arrpc";
-    # Release commits are not tagged
-    # release: 3.4.0
-    rev = "cca93db585dedf8acc1423f5e2db215de95c4c3b";
-    hash = "sha256-SeegrCgbjfVxG/9xfOcdfbVdDssZOhjBRnDipu6L7Wg=";
+    rev = version;
```

```suggestion
    tag = version;
```
This does the "refs/tags" thing automatically. #355973